src/bin/travis/testSbt.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
FWIW my use of `/bin/bash` previously was intentional as all of the Travis examples do it that way - I just assumed it was a Travis best practice. OTOH presumably this works in Travis so eh